app/src/main/java/com/example/homework1/Console.kt

```diff
@@ -1,4 +1,65 @@
 package com.example.homework1
 
 class Console {
+
+    object calculator {
+        fun console() {
+            var onOff = true
+            var result: Double? = null
+            var operation: String? = null
+            var currentResult: Double? = null
+
+
+            while (onOff) {
+                println("Enter the number")
+                val number = (readLine()?.toDouble() ?: 0) as Double
+                if (currentResult == null) {
+                    currentResult = number
+                } else {
+                    when (operation) {
+                        "-" -> {
+                            result = currentResult - number
+                            currentResult = result
+                        }
+                        "+" -> {
+                            result = currentResult + number
+                            currentResult = result
+                        }
+                        "*" -> {
+                            result = currentResult * number
+                            currentResult = result
+                        }
+                        "/" -> {
+                            result = currentResult / number
+                            if((number%1).equals(0.0)) {
```

Comment:
а зачем так сложно? % это остаток от деления
в твоем случае остаток от деления number на 1, то ест если будет 0, то ты и получишь 0, а если другое число то оно же и будет
```suggestion
                            if(number == 0) {
```